app/src/main/kotlin/cz/covid19cz/erouska/ui/permissions/BasePermissionsFragment.kt

```diff
@@ -72,23 +72,30 @@ open class BasePermissionsFragment<T : BasePermissionsVM>(@LayoutRes layout: Int
         }
 
         MaterialAlertDialogBuilder(context)
-            .setCustomTitle(layout)
-            .setMessage(getString(R.string.permission_rationale_body))
-            .setPositiveButton(getString(R.string.permission_rationale_settings))
-            { dialog, _ ->
-                dialog.dismiss()
-                requireContext().openPermissionsScreen()
-            }
-            .setNegativeButton(getString(R.string.permission_rationale_dismiss))
-            { dialog, _ -> dialog.dismiss() }
-            .show()
+                .setCustomTitle(layout)
+                .setMessage(getString(R.string.permission_rationale_body))
+                .setPositiveButton(getString(R.string.permission_rationale_settings))
+                { dialog, _ ->
+                    dialog.dismiss()
+                    requireContext().openPermissionsScreen()
+                }
+                .setNegativeButton(getString(R.string.permission_rationale_dismiss))
+                { dialog, _ -> dialog.dismiss() }
+                .show()
     }
 
     private fun enableLocation() {
         requireContext().openLocationSettings()
     }
 
     private fun enableBluetooth() {
-        requestEnableBt()
+        MaterialAlertDialogBuilder(context)
```

Comment:
Zrovna tenhle dialog je systémový - vytváří ho Google API. Odeber ho z kódu.

Reply:
![Screenshot_20200715-124549](https://user-images.githubusercontent.com/19397052/87536652-b9ef8700-c699-11ea-9b10-10e7fea9c7d0.png)
Zde jde o tento dialog s rationale, proč chce aplikace zapnout BT, viz screen D1 ve Figma.
Další dialog s "Aplikace eRouška chce zapnout Bluetooth" (nebo tak něco) se samozřejmě dále zobrazuje pomocí Google API jako bylo doteď.

Reply:
Ono to bude fungovat jinak - naše aplikace nebude mít vůbec přístup k Bluetooth, technicky to scanování dělají Google Play Services. A Google Play Services poskytují tenhle dialog. Nainstaluj si tu sample aplikaci a zkus si to aktivovat, ať vidíš ten flow.

Reply:
V sample appce mi to bohužel háže generický errory (bavíme se o té exposure-notifications API?), takže jsem si to flow nevyzkoušel, ale prošel jsem dokumentaci a asi chápu, o dialog pro povolení scanování a broadcastu... Tento dialog prozatím tedy odebírám